### ISSUE TITLE
alphashapes & n<4

### ISSUE THREAD

**Issue by ljwolf** [pysal/libpysal#111]
in theory, the "delaunay" triangulation for cases where the number of points is less than 4 is still "known." N=3 is the triangle, n=2 is the line, and n=1 is the point. 

as it stands, `Qhull` will error out when passed a collection with less than four points. It'd be nice if we checked this & returned something sane in these cases, rather than bailing out from qhull.

**Comment by ljwolf** [comment]
I think this is sufficient:

```python
def safe_ashape(xys, **ashape_kws):
    if xys.shape[0] < 4:
        return ops.cascaded_union([geom.Point(xy) for xy in xys]).convex_hull
    return alpha_shape(xys, **ashape_kws)
```

**Comment by darribas** [comment]
I've ran into this issue too. I think that solution might do it. Just a question, would that return a polygon object?

**Comment by jGaboardi** [comment]
Addressed in #115
